### PR TITLE
Add a way to post attributes by email

### PR DESCRIPTION
**Pull request by maccman**

### CODE REVIEW

File: lib/chartmogul/v1/customers.rb [hunk 1]
```diff
@@ -60,6 +64,30 @@ def update(customer_id, attributes = [])
         end
       end
 
+      class EmailAttributes < Customers
+        # Public: Create Custom Attributes for a Customer
+        #
+        # See: https://dev.chartmogul.com/docs/create-custom-attributes-for-customer
```
Comment (avokhmin) [inline lib/chartmogul/v1/customers.rb:70]: Misprint, should be `https://dev.chartmogul.com/docs/create-custom-attributes-for-customer-with-email`

File: lib/chartmogul/v1/customers.rb [hunk 2]
```diff
@@ -60,6 +64,30 @@ def update(customer_id, attributes = [])
...
+        #
+        # Examples
+        #
+        #   client.customers.attributes.create 'alex@clearbit.com', [
+        #     { type: 'String',  key: 'bar', value: 'baz' },
+        #     { type: 'Integer', key: 'qux', value: 1 }
+        #   ]
+        #
+        # Returns the instance of Chartmogul::V1::Request.
+        def create(email, attributes = [])
```
Comment (avokhmin) [inline lib/chartmogul/v1/customers.rb:83]: maybe better to add `def create_for_email(email, attributes = [])`

or

```
def create(customer_id: nil, email: nil, attributes = [])
  body = { custom: attributes }
  body.merge!(email: email) if email
  url = if customer_id
          "#{BASE_URI}/#{customer_id}/attributes/custom"
        else
          "#{BASE_URI}/attributes/custom"
        end
  Chartmogul::V1::Request.new(url,
    body:    body,
    method:  :post,
    userpwd: client.userpwd,
  )
end
```

**Comment by avokhmin** [comment]
@maccman, can you add a specs too please.

**Comment by recurser** [comment]
Closing due to inactivity - @maccman please feel free to re-open if you'd like.

**Pull request closed without merging**